### PR TITLE
Move to django.

### PR DESCRIPTION
**Pull request by jkp85**

### CODE REVIEW

File: billing/models.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+class BillingAddress(models.Model):
+    address = models.CharField(max_length=255)
+    city = models.CharField(max_length=255)
+    state_province = models.CharField(max_length=100)
+    zip_code = models.CharField(max_length=11)
+    created_at = models.DateTimeField(blank=True, null=True)
+
+    class Meta:
+        db_table = 'billing_address'
```
Comment (rkorkosz) [inline billing/models.py]: Good point

File: servers/spawners/dummyspawner.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
+from hubserver.app.servers.spawner import ServerSpawner
```
Comment (rkorkosz) [inline servers/spawners/dummyspawner.py]: This is usefull for testing.

File: servers/spawners/dockerspawner.py [hunk 3]
```diff
@@ -0,0 +1,267 @@
...
+from servers.spawner import ServerSpawner
+
+logger = logging.getLogger(__name__)
+
+
+class DockerSpawner(ServerSpawner):
+    def __init__(self, server):
+        super().__init__(server)
+        self.client = from_env()
+        self.container_port = self.server.server.environment_type.container_port
```
Comment (rkorkosz) [inline servers/spawners/dockerspawner.py]: This is due to different handling of one to one relations by django comparing to sqlalchemy. It should be refactored.

File: billing/models.py (hunk 1 above)
Comment (jkp85) [inline billing/models.py]: Good point

File: servers/spawners/dummyspawner.py (hunk 2 above)
Comment (jkp85) [inline servers/spawners/dummyspawner.py]: This is usefull for testing.

File: servers/spawners/dockerspawner.py (hunk 3 above)
Comment (jkp85) [inline servers/spawners/dockerspawner.py]: This is due to different handling of one to one relations by django comparing to sqlalchemy. It should be refactored. In sqlalchemy it was self.server and we use concrete servers like models/workspaces and those model have field server

**Pull request merged**